ConfigurationSystem/Client/Helpers/Resources.py

```diff
@@ -295,3 +296,83 @@ def getFilterConfig(filterID):
   :params filterID: string representing a filter identifier.
   """
   return gConfig.getOptionsDict('Resources/LogFilters/%s' % filterID)
+
+
+def getProxyProviders():
+  """ Return list of all proxy provider names defined in the CS
+
+      :return: S_OK(list)/S_ERROR
+  """
+  return gConfig.getSections('%s/ProxyProviders' % gBaseResourcesSection)
+
+
+def getProxyProviderOption(proxyProvider, option):
```

Comment:
This method is unused

Reply:
This logic moved to new method getInfoAboutProviders